src/Permissions/Permission.php

```diff
@@ -1,56 +1,42 @@
 <?php
 
+declare(strict_types=1);
+
 namespace LaravelDoctrine\ACL\Permissions;
 
 use Doctrine\ORM\Mapping as ORM;
 use LaravelDoctrine\ACL\Contracts\Permission as PermissionContract;
 
-/**
- * @ORM\Entity
- */
+#[ORM\Entity]
 class Permission implements PermissionContract
 {
-    /**
-     * @ORM\Id
-     * @ORM\Column(type="integer")
-     * @ORM\GeneratedValue(strategy="AUTO")
-     */
-    protected $id;
-
-    /**
-     * @ORM\Column(type="string")
-     */
-    protected $name;
-
-    /**
-     * @param $name
-     */
-    public function __construct($name)
+    #[ORM\Id]
+    #[ORM\Column(type: 'integer')]
+    #[ORM\GeneratedValue(strategy: 'AUTO')]
+    protected int $id;
+
+    #[ORM\Column(type: 'string')]
+    protected string|null $name;
+
+    public function __construct(string $name)
     {
         $this->name = $name;
     }
 
-    /**
-     * @return mixed
-     */
-    public function getId()
+    public function getId(): int
     {
         return $this->id;
     }
 
-    /**
-     * @return string
-     */
-    public function getName()
+    public function getName(): string
```

Comment:
returns `string|null`

Reply:
`$name` can be `string|null` so the return of `getName()` must be `string|null` OR change `$name` to be `string`.